paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerViewModel.kt

```diff
@@ -37,6 +44,19 @@ internal class FlowControllerViewModel(
         restartSession()
     }
 
+    class Factory(
+        private val statusBarColor: () -> Int?
+    ) : ViewModelProvider.Factory {
+        @Suppress("UNCHECKED_CAST")
+        override fun <T : ViewModel> create(modelClass: Class<T>, extras: CreationExtras): T {
+            return FlowControllerViewModel(
+                application = extras.requireApplication(),
+                handle = extras.createSavedStateHandle(),
+                statusBarColor = statusBarColor,
+            ) as T
+        }
+    }
```

Comment:
Haven't been able to find a good way to not inject this through the view model when creating the `ConfirmationHandler`.